### DISCUSSION TITLE
[Bug]: Terragenesis Mountainous setting does nothing for Tropic landscape

### DISCUSSION THREAD

**Discussion by supermop** [OpenTTD/OpenTTD#11734] [Features]
### Version of OpenTTD

1.12.0

### Expected result

Tropic Landscape generation in Terra Genesis can be more or less hilly depending on options selected.

### Actual result

Even with mountainous or alpinist, regardless of variety distribution setting, tropic maps are very flat, with no real hilly areas.  It's fine that the different climates are distinct in their topography, but it would be nice if the mountainous setting made at least some hilly areas, so we could quickly generate different types of tropic maps without needing to use heightmaps or the scenario editor.

I know this was discussed at some length a few years ago, but it seems nothing came of that?

### Steps to reproduce

1. Generate a new map with tropic landscape, terragenesis, and either mountainous or alpinist setting for the terrain type
2. Observe your disappointingly non-mountainous landscape
3. Generate a map with 'flat' type, and observe that it looks fairly similar
4. Repeat above steps with temperate or sub-arctic and observe that the setting has a noticeable effect.

**Comment by 2TallTyler** [comment]
There's some discussion about how terragen differs between climates, and how it might be separated from the current hardcoded behavior, in #9634.

**Comment by 2TallTyler** [comment]
Implemented in #14989.
